app/models/manageiq/providers/azure/cloud_manager/event_catcher/runner.rb

```diff
@@ -49,6 +45,8 @@ def reset_event_monitor_handle
 
   def filtered?(event)
     event_type = parse_event_type(event)
-    filtered_events.include?(event_type)
+    filtered_events.include?(event_type).tap do |result|
+      _log.info "#{log_prefix} Skipping filtered Azure event #{event_type} for #{event["resourceId"]}" if result
+    end
```

Comment:
TBH I'd never seen this before as a way to control a return value with a side effect, either (the boolean isn't forced, either, I mean - `include?` returns what we want already). I don't know for sure but I don't think this is 'common style' simply because if the code calls for a more event driven approach your code would be structured with something more attuned to that behavior (like a series of callbacks, whatever) in which case you're not inserting side behavior on a return value (logging, in this case)